compose/Makefile

```diff
@@ -164,6 +164,33 @@ test-at-check: test-at-build  ## AMAUAT: test browsers.
 	$(call compose_amauat, \
 		run --rm --no-deps archivematica-acceptance-tests /home/archivematica/acceptance-tests/simplebrowsertest.py)
 
+define AT_HELP
+
+   Archivematica acceptance tests (Listing).
+
+   The most effective way to run these tests is to run them by tag. For
+   example:
+
+      $ make test-at-behave TAGS=black-box BROWSER=Firefox
+
+   Commonly used acceptance tests in the Archivematica suite:
+
+      * aip-encrypt 	   :Tests the encryption of AIPs.
```

Comment:
Indentation issue here.

Reply:
Can you elaborate on the indentation issue? This is how I wrote it.

Reply:
There's one tab there messing with my terminal.

![image](https://user-images.githubusercontent.com/606459/65992627-65e61f80-e444-11e9-9e87-4a1b7fb57c12.png)

![image](https://user-images.githubusercontent.com/606459/65992542-3fc07f80-e444-11e9-82e3-f2577d7dd06d.png)